libs/community/langchain_community/utilities/yandex_search.py

```diff
@@ -0,0 +1,225 @@
+import warnings
+import xml.etree.ElementTree as ET
```

Comment:
Please use [defusexml](https://pypi.org/project/defusedxml/) rather than built-in XML. https://docs.python.org/3/library/xml.etree.elementtree.html

Reply:
done